test/test_transforms.py

```diff
@@ -2090,5 +1923,193 @@ def test_normalize_3d_tensor():
     torch.testing.assert_close(target, result2)
 
 
+class TestAffine:
+
+    input_img = np.zeros((40, 40, 3), dtype=np.uint8)
+    cnt = [20, 20]
+
+    @pytest.mark.parametrize("pt", [(16, 16), (20, 16), (20, 20)])
+    @pytest.mark.parametrize("i", range(-5, 5))
+    @pytest.mark.parametrize("j", range(-5, 5))
+    def test_make_img_affine(self, pt, i, j):
+        self.input_img[pt[0] + i, pt[1] + j, :] = [255, 155, 55]
+
+    def test_affine_translate_seq(self):
+        with pytest.raises(TypeError, match=r"Argument translate should be a sequence"):
+            F.affine(self.input_img, 10, translate=0, scale=1, shear=1)
+
+    @pytest.fixture
+    def PIL_Image(self):
+        return F.to_pil_image(self.input_img)
+
+    def _to_3x3_inv(self, inv_result_matrix):
+        result_matrix = np.zeros((3, 3))
+        result_matrix[:2, :] = np.array(inv_result_matrix).reshape((2, 3))
+        result_matrix[2, 2] = 1
+        return np.linalg.inv(result_matrix)
+
+    def _test_transformation(self, a, t, s, sh, PIL_Image):
```

Comment:
here and in the rest of the code, let's take this opportunity to use better names: angle, translate, scale, shear, etc.

Reply:
okay, will do